jbmc/src/java_bytecode/simple_method_stubbing.cpp

```diff
@@ -222,10 +222,15 @@ void java_simple_method_stubst::create_method_stub(symbolt &symbol)
 void java_simple_method_stubst::check_method_stub(const irep_idt &symname)
 {
   const symbolt &sym = *symbol_table.lookup(symname);
-  if(
-    !sym.is_type && sym.value.id() == ID_nil && sym.type.id() == ID_code &&
-    // Don't stub internal locking primitives:
-    sym.name != "java::monitorenter" && sym.name != "java::monitorexit")
+  if(!sym.is_type && sym.value.id() == ID_nil &&
+    sym.type.id() == ID_code &&
+    // do not stub internal locking calls as 'create_method_stub'
+    // does not automatically create the appropriate symbols for
+    // the formal parameters.
```

Comment:
This is only half of the reason. Add `, and symex will crash when a function has no formal parameters`, or whatever the reason is.

Reply:
done